### ISSUE TITLE
Support for Django 3.X

### ISSUE THREAD

**Issue by pallavg** [bartTC/django-wakawaka#18]
I am using Django 3.0.2 using python 3.7.6. I installed django-wakawaka from pip. When I ran `python manage.py check`, I got the following error:

```
(base) C:\Users\pgupta6\Desktop\PyCharmProjects\personal-work-repo\python\src\tvpv_portal>python manage.py check
Traceback (most recent call last):
  File "manage.py", line 30, in <module>
    main()
  File "manage.py", line 26, in main
    execute_from_command_line(sys.argv)
  File "C:\Anaconda3\lib\site-packages\django\core\management\__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "C:\Anaconda3\lib\site-packages\django\core\management\__init__.py", line 377, in execute
    django.setup()
  File "C:\Anaconda3\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Anaconda3\lib\site-packages\django\apps\registry.py", line 114, in populate
    app_config.import_models()
  File "C:\Anaconda3\lib\site-packages\django\apps\config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\Anaconda3\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Anaconda3\lib\site-packages\wakawaka\models.py", line 5, in <module>
    from django.utils.six import python_2_unicode_compatible
ModuleNotFoundError: No module named 'django.utils.six'
```

It looks like this module has been dropped from Django 3.X. Is it possible to get a fix for this?

Probably replace `@python_2_unicode_compatible` with `@six.python_2_unicode_compatible`. See https://six.readthedocs.io/#six.python_2_unicode_compatible

**Comment by pallavg** [comment]
PR potentially addressing this is submitted for review/acceptance.

**Comment by bartTC** [comment]
Thank you for the PR. I'm going to drop Py2.7 and with that six completely.

**Comment by pallavg** [comment]
Great.

**Comment by bartTC** [comment]
Fixed in v1.2 https://pypi.org/project/django-wakawaka/
